HCK19_2024_Janelia_NDRH/README.md

```diff
@@ -0,0 +1,127 @@
+and read more about this feature event.
+-->
+
+Example projects include but are not limited to:
+* Determine whether your result is present in another species or brain area.
+* Showcase the capabilities of your tool or analysis on existing data from another lab.
+* Explore follow-up questions to a study.
+* Create educational content that uses open data.
+
+<!-- 
+Following the event, participants will be invited to apply for a Kavli Foundation Neurodata Discovery Award, which 
+awards $50,000 (USD) of funding to continue data reanalysis projects that come out of the NeuroDataReHack event. Details about the Kavli Foundation Neurodata Discovery Award will be provided closer to the event.
+-->
+This event will primarily focus on analyzing existing data in NWB and on DANDI, **not** converting data to NWB. If 
+you are interested in learning how to convert data, consider signing up for an NWB User Days event.
+
+## Eligibility
+
+This course is intended for PhD students, postdoctoral researchers, principal investigators, or similar.
+Applicants should have basic programming experience in Python or MATLAB and experience with neurophysiology research.
+
+
+## Application
+
+Space for the event is limited. Apply to attend NeuroDataReHack 2024 [here](https://forms.gle/tzumwuVgrkdD4zm36). 
+
+* Application deadline: March 1
+* Notification of admission decisions: April 1
+
+## Logistics
+
+Thanks to the generous sponsorship of The Janelia Research Campus, this event will be free to participants:
+* There is no registration or application fee.
+* Participants will be provided a private room at Janelia Research Campus for the duration of this event, 
+  checking in on Sunday, July 7, and checking out on Friday, July 12 or Saturday, July 13.
+* Breakfast, lunch, dinner, and coffee breaks will be provided.
+* You will be responsible for booking your own flight and transportation to Janelia Research Campus.
+* You will be responsible for determining visa and/or healthcare requirements for travel to the United States.
+
+## Organizing Committee
+
+**Program chairs:**
+* Benjamin Dichter, CatalystNeuro
+* Oliver Rübel, Lawrence Berkeley National Laboratory
+* Ryan Ly, Lawrence Berkeley National Laboratory
+
+## Resources
+
+Resources will be posted here to help participants prepare for the event.
+
+* A report of the first NeuroDataReHack event: ([PDF](../HCK14_2022_Seattle_RH/report/Report_Neurodata_Rehack_v2.pdf))
+* A report of the second NeuroDataReHack event: ([PDF](../HCK16_2023_Granada_RH/report/Report__NeuroDataReHack_2023.pdf))
+* Recordings of talks will be made available on the [NWB Youtube channel](https://www.youtube.com/channel/UCfD_mU-EFz135a9TpNFJP5A).
```

Comment:
```suggestion
```
Remove this? We did not make any talks available last time. Do we want to make them available this time?

Reply:
Yes, because it will be at Janelia so they will record the talks by default. This will be a good opportunity for us to build up a library of recorded lectures, as opposed to other venues that are often not set up to do that